js/carousel.js

```diff
@@ -0,0 +1,16 @@
+let slideIndex = 1;
+showSlides(slideIndex);
+
+function plusSlides(n) {
+  showSlides(slideIndex += n);
+}
+
+function showSlides(n) {
+  let slides = document.getElementsByClassName("mySlides");
+  if (n > slides.length) {slideIndex = 1}    
+  if (n < 1) {slideIndex = slides.length}
```

Comment:
You can avoid brackets if you are writing the if's block in one line, but you can leave it as it is, that's okay too.
```suggestion
  if (n > slides.length) slideIndex = 1    
  if (n < 1) slideIndex = slides.length
```